public/app/listeners/loadDataset.js

```diff
@@ -24,29 +25,35 @@ const createNewDataset = ({ name, filepath }) => {
     id: fileId,
     name,
     filepath: destPath,
+    description,
     size: sizeInKiloBytes,
     createdAt,
     lastModified,
   };
 };
 
 const loadDataset = (mainWindow, db) => async (event, args) => {
-  const { fileLocation } = args;
-
-  // filename without extension
-  const filename = path
-    .basename(fileLocation)
-    .slice(0, -path.extname(fileLocation).length);
+  const { fileLocation, fileDescription } = args;
```

Comment:
Just thinking, we might want some check on `fileLocation` such as whether the file really exists.

Reply:
Good call again!